kubernetes-manifests/emailservice.yaml

```diff
@@ -34,8 +34,10 @@ spec:
         env:
         - name: PORT
           value: "8080"
-        - name: ENABLE_PROFILER
```

Comment:
if the default in the release was "profiler disabled", might be wise to uncomment that in the final result.